scripts/generate.sh

```diff
@@ -1,7 +1,7 @@
 . $(dirname $0)/common.sh
 
 bundle --gemfile=$SCRIPTS_ROOT/generator/Gemfile
-$SCRIPTS_ROOT/generator/update_schema
+$SCRIPTS_ROOT/generator/update_schema $1
```

Comment:
I'd like to do this change in another PR and fix all sh linting issues at the same time.